Documentation/Architecture/CrossPlatform/DeterminingPlatformCapabilities.md

```diff
@@ -0,0 +1,60 @@
+# Determining platform capabilities
+
+A common question asked of the MRTK involves knowing which specific device (ex: Microsoft HoloLens 2) is being
+used to run an application. The goal behind that knoweldge generally centers around knowing whether or not a
+particular feature (ex: articulated hands) is available.
+
+Mixed reality devices are evolving rapidly and any attempt at capturing a table mapping features to specific
+hardware will quickly become untennable.
+
+## Checking for platform capabilities
+
+To help answer the root question of whether or not an application can leverage a specific feature, the Mixed Reality
+Toolkit provides the [IMixedRealityCapabilityCheck](xref:Microsoft.MixedReality.Toolkit.IMixedRealityCapabilityCheck)
+interface.
+
+### Querying for a capability
+
+The most common pattern for determining support for a specific capability is to query the service.
```

Comment:
Query what service? How do I know which service to query? There are N number of MRTK services - so as a consumer do I have to know both what service to query and what questions they can answer? Or can I just ask a single thing what the entire system's capabilities are?

Reply:
Ah, I see now - i feel like there should be some reference here so that people can have a more immediate link